build.gradle.kts

```diff
@@ -35,7 +35,7 @@ plugins {
     idea
     kotlin("jvm") version "1.3.30"
     id("org.jetbrains.intellij") version "0.4.8"
-    id("org.jetbrains.grammarkit") version "2018.2.2"
+    id("org.jetbrains.grammarkit") version "2019.1"
```

Comment:
It doesn't work with CLion because of https://github.com/JetBrains/gradle-grammar-kit-plugin/issues/17
It's already fixed but a new version with the fix is not published yet

Reply:
Oops. I'll revert it.

Reply:
Reverted